package.json

```diff
@@ -5,66 +5,83 @@
+    "init": "pnpm install && pnpm build",
+    "init17": "pnpm install && pnpm build:cli && npx ts-node tools/test/scripts/install-react17.ts && pnpm build",
     "create-package": "lg create",
-    "prebuild": "yarn build:cli",
+    "prebuild": "pnpm build:cli",
     "build": "turbo run build tsc",
     "build:cli": "turbo run build tsc --filter=@lg-tools/cli",
     "build-storybook": "npx storybook build",
     "build:tsc": "turbo run tsc",
     "chromatic": "npx chromatic",
     "clean": "npm-run-all --parallel clean:*",
-    "clean:builds": "rm -rf {packages,tools,chat,charts}/*/{dist,tsconfig.tsbuildinfo,stories.js,.turbo}",
-    "clean:cache": "rm -rf .turbo {cache}",
-    "clean:modules": "rm -rf node_modules {packages,tools,chat,charts}/*/node_modules",
-    "clean:ts": "rm -rf {packages,tools,chat,charts}/*/{dist/**/*.ts,dist/**/*.ts.map,tsconfig.tsbuildinfo,ts-trace,.turbo}",
+    "clean:builds": "pnpm recursive exec -- rm -rf ./{dist,tsconfig.tsbuildinfo,stories.js}",
+    "clean:modules": "pnpm recursive exec -- rm -rf node_modules; rm -rf node_modules",
     "fix": "lg lint --fix",
     "link": "lg link",
     "lint": "lg lint",
-    "prepublishOnly": "yarn build && turbo run docs",
-    "publish": "yarn changeset publish --public",
+    "prepublishOnly": "pnpm run build && turbo run docs",
+    "publish": "pnpm changeset publish --public",
     "slackbot": "lg slackbot release",
     "start": "npx storybook dev -p 9001 --no-version-updates",
     "serve": "npx http-server storybook-static -c5",
     "test": "lg test",
     "unlink": "lg unlink",
     "validate": "lg validate",
-    "watch": "npx nodemon --watch packages/ -e tsx,ts --exec 'yarn build-storybook --test'"
+    "watch": "npx nodemon --watch packages/ -e tsx,ts --exec 'pnpm run storybook build --test'"
   },
   "devDependencies": {
     "@actions/core": "^1.10.1",
     "@babel/core": "7.24.3",
     "@changesets/cli": "^2.26.0",
-    "@lg-tools/cli": "*",
-    "@lg-tools/storybook-addon": "*",
+    "@leafygreen-ui/testing-lib": "workspace:^",
+    "@lg-tools/build": "workspace:^",
+    "@lg-tools/cli": "workspace:^",
+    "@lg-tools/lint": "workspace:^",
+    "@lg-tools/meta": "workspace:^",
+    "@lg-tools/storybook-addon": "workspace:^",
+    "@lg-tools/storybook-utils": "workspace:^",
+    "@lg-tools/test": "workspace:^",
+    "@storybook/react": "^7.6.20",
+    "@svgr/babel-plugin-replace-jsx-attribute-value": "8.0.0",
+    "@testing-library/react": "^12.0.0 || ^13.1.0 || ^14.0.0",
+    "@testing-library/react-hooks": ">=3.7.0",
+    "@testing-library/user-event": "13.5.0",
+    "@types/cross-spawn": "6.0.2",
+    "@types/fs-extra": "11.0.1",
+    "@types/jest": "^29.5.12",
     "@types/lodash": "^4.14.170",
+    "@types/node": "^20.12.5",
     "@types/react": "18.2.23",
     "@types/react-dom": "18.2.8",
+    "@types/react-transition-group": "^4.4.10",
     "chromatic": "^11.0.0",
+    "cross-spawn": "7.0.3",
```

Comment:
Curious why we have a lot of these installed locally. I don't believe we need:
`cross-spawn`, `fs-extra`, `eslint`, `node-fetch`, `npm-package-json-lint`, `prettier`, `ts-node`
(and their associated `@types/*` packages)

Reply:
(we might need prettier for now, but we should get rid of the global dep eventually)

Reply:
Got it. Those should be moved into lint itself eventually